sql/src/main/java/io/crate/execution/engine/collect/collectors/RemoteCollector.java

```diff
@@ -80,6 +81,9 @@ public RemoteCollector(UUID jobId,
         this.localNode = localNode;
         this.remoteNode = remoteNode;
 
+        // TODO: check if profiling is required
```

Comment:
I think it's okay to have this as false - maybe change the comment to explain that the duration for the `remoteCollect` will be accounted for by the `JobCollectContext` which contains *this* `RemoteCollector`